input/elasticapm/internal/modeldecoder/modeldecodertest/populator.go

```diff
@@ -263,6 +265,10 @@ func SetZeroStructValue(i interface{}, callback func(string)) {
 // that values are equal to expected values
 func AssertStructValues(t *testing.T, i interface{}, isException func(string) bool,
 	values *Values) {
+	if true {
+		// TODO FIX no op protobuf support
+		return
```

Comment:
I'm not particularly comfortable with this. If we're merging without these tests, please open an issue so we can get it back in ASAP. Perhaps we should consider getting rid of the reflection approach.

Reply:
I removed the TODO and updated tests to remove/replace the AssertStructValues usage where I thought it made sense: https://github.com/axw/apm-data/commit/e3a387cc21fad079d6bf170164dcd8c9255e6d12. Please have a look and feel free to cherry-pick that if you think it looks sense.

Reply:
Thanks for this! I cherry-picked your commit!